### ISSUE TITLE
How to enable automatic logging to TensorBoard or Weights & Biases during fine-tuning?

### ISSUE THREAD

**Issue by LeMoussel** [amazon-science/chronos-forecasting#386]
Hi! 

First of all, thanks a lot for releasing Chronos-2 — it’s an amazing project.

I’m currently fine-tuning Chronos-2 using the Chronos2Pipeline API:
```python
from chronos import BaseChronosPipeline, Chronos2Pipeline

pipeline = BaseChronosPipeline.from_pretrained("amazon/chronos-2", device_map="cuda")

finetuned = pipeline.fit(
    inputs=train_inputs,
    prediction_length=13,
    num_steps=50,
    learning_rate=1e-5,
    batch_size=32,
    logging_steps=10,
)
```
I’m trying to figure out whether it is possible to log training metrics (loss, learning rate, etc.) automatically to TensorBoard or Weights & Biases (W&B).

Is there a built-in way to attach a logger (TensorBoardLogger, WandbLogger, custom callbacks, etc.) to `Chronos2Pipeline.fit()`?
If not currently supported, is there a recommended workaround or extension point inside the library?
Are the internal training loops available somewhere so that we can inject custom logging manually?

I didn’t find documentation about logging or callbacks, so I want to make sure I’m not missing something.

Thanks a lot for your help — and for the amazing work on Chronos!

**Comment by shchur** [comment]
You should be able to enable arbitrary logging backends using
https://github.com/amazon-science/chronos-forecasting/blob/67a6f91aec23ddbd8a256270af9491a4ad1b570d/src/chronos/chronos2/pipeline.py#L143
